app/src/main/java/com/asksira/imagepickersheetdemo/MainActivity.java

```diff
@@ -27,13 +31,21 @@ protected void onCreate(Bundle savedInstanceState) {
         ivImage4 = findViewById(R.id.iv_image4);
         ivImage5 = findViewById(R.id.iv_image5);
         ivImage6 = findViewById(R.id.iv_image6);
+
+        String timeStamp = new SimpleDateFormat("yyyyMMdd_HHmmss").format(Calendar.getInstance().getTime());
+        String imageFileName = "Repsly1" + "/JPEG_" + timeStamp + "_";
+        File storageDir = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES);
```

Comment:
Kako će ti se ovo slagati s novim promjenama koje moramo raditi za odabir da nam se spremaju slike u public folder ili ne?

Reply:
ovo je main activity, to testiram samo u tom projektu, nema veze sa ovim promjenama sto su radene u BsImagePicker